src/GovUk.Education.ExploreEducationStatistics.Data.Processor/ProcessorStatistics.cs

```diff
@@ -1,21 +1,22 @@
+#nullable enable
 using System.Collections.Generic;
 using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
 
 namespace GovUk.Education.ExploreEducationStatistics.Data.Processor
 {
     public class ProcessorStatistics
     {
-        public int FilteredObservationCount { get; set; }
-        public int RowsPerBatch { get; set; }
-        public int NumBatches { get; set; }
-        public HashSet<GeographicLevel> GeographicLevels { get; set; }
+        public int TotalRows { get; }
```

Comment:
Have now renamed it to `ImportableRowCount` to reflect more accurately what it's a count of.
Name was originally `FilteredObservationCount` but I think 'rows' is more appropriate here to importing data.

The `rows` and `totalRows` thing is a mess but hopefully this will be addressed by a combination of EES-2006 and EES-1810.